src/Umbraco.Infrastructure/BackgroundJobs/Jobs/ReportSiteJob.cs

```diff
@@ -19,25 +20,37 @@ public class ReportSiteJob : IRecurringBackgroundJob
     // No-op event as the period never changes on this job
     public event EventHandler PeriodChanged
     {
-        add { } remove { }
+        add { }
+        remove { }
     }
 
-    private static HttpClient _httpClient = new();
-
     private readonly ILogger<ReportSiteJob> _logger;
-
     private readonly ITelemetryService _telemetryService;
-private readonly IJsonSerializer _jsonSerializer;
+    private readonly IJsonSerializer _jsonSerializer;
+    private readonly IHttpClientFactory _httpClientFactory;
 
+    [Obsolete("Use the constructor with IHttpClientFactory instead.")]
     public ReportSiteJob(
         ILogger<ReportSiteJob> logger,
         ITelemetryService telemetryService,
         IJsonSerializer jsonSerializer)
+        : this(
+            logger,
+            telemetryService,
+            jsonSerializer,
+            StaticServiceProvider.Instance.GetRequiredService<IHttpClientFactory>())
+    { }
+
+    public ReportSiteJob(
+        ILogger<ReportSiteJob> logger,
+        ITelemetryService telemetryService,
+        IJsonSerializer jsonSerializer,
+        IHttpClientFactory httpClientFactory)
     {
         _logger = logger;
         _telemetryService = telemetryService;
         _jsonSerializer = jsonSerializer;
-        _httpClient = new HttpClient();
```

Comment:
Using a static `HttpClient` without setting the `PooledConnectionLifetime` can cause issues with DNS changes: https://learn.microsoft.com/en-us/dotnet/fundamentals/networking/http/httpclient-guidelines. Using `IHttpClientFactory` instead will make sure connection pooling and lifetime management are done automatically 👍🏻